### DISCUSSION TITLE
Find and replace?

### DISCUSSION THREAD

**Discussion by lucijaff** [usebruno/bruno#3446] [Q&A] [answered]
I can't seem to find a way to find and replace in the request body in Bruno, is this not built in?

**Comment by lucijaff** [comment, marked as answer]
Good thing I asked (searching threads didn't help). It's ⌘⌥F on macOS

**Comment by fievelk** [comment]
Hi, is there a way to find and replace all occurrences of a specific variable (in all requests)?
For example: all my requests are using a variable called `baseUrl`. How can I replace this to be something else (e.g. `baseUrlXyz`)?
